### DISCUSSION TITLE
Please provide way how to generate credentials

### DISCUSSION THREAD

**Discussion by JaLe29** [html2app/discussions#4] [Q&A] [answered]
Hi, 

I would like to create build for store, but I don't know where/how to generate credentials.

![image](https://user-images.githubusercontent.com/22296816/203383564-e59951a0-7895-41d2-a55a-a7487cedcf5d.png)

**Comment by yandeu** [comment, marked as answer]
I will add a generator. But for now, please use this guide:
https://ionic.io/docs/appflow/package/credentials

**Reply by JaLe29** [reply]
Thank you!
